tests/Builders/Types/TypeTagFlagTest.php

```diff
@@ -19,11 +20,10 @@ public function test__construct()
         $this->assertSame($type->getValue(), $type->export(1));
     }
 
-    /**
-     * @expectedException \InvalidArgumentException
-     */
```

Comment:
The `@expectedException` comment annotation is deprecated on `PHPUnit 8.x` and it will be removed on `PHPUnit 9.x`.

It's time to remove this comment annotation. And using `$this->expectException` method instead.